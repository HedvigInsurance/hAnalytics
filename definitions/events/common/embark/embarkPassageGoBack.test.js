const validateEvent = require("commons/validateEvent.js");

test("validate", () => {
  const event = validateEvent(__filename, {
    storyName: "Test",
    passageName: "Passage",
  });

  expect(event).toMatchSnapshot();
});
