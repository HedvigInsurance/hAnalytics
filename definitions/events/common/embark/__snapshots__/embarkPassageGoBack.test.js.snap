// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate 1`] = `
Object {
  "name": "embark_passage_go_back",
  "variables": Object {
    "originated_from_embark_story": "Test",
    "passage_name": "Passage",
  },
}
`;
